src/main/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltration.java

```diff
@@ -114,6 +114,7 @@ public final class VariantFiltration extends VariantWalker {
     public static final String CLUSTER_WINDOW_SIZE_LONG_NAME = "cluster-window-size";
     public static final String MASK_EXTENSION_LONG_NAME = "mask-extension";
     public static final String MASK_NAME_LONG_NAME = "mask-name";
+    public static final String MASK_DESCRIPTION_LONG_NAME = "mask-desc";
```

Comment:
I might spell out `mask-description` for this argument name, since the other mask arguments don't have abbreviations